apps/frontend/src/app/components/footer/footer.component.html

```diff
@@ -17,32 +17,49 @@ <h4>Linkek</h4>
       <li><a href="https://verseghy-gimnazium.net/">Vissza a régi oldalra</a></li>
     </ul>
   </div>
-  <div class="footerblock subscribe">
-    <h4>Hírlevél</h4>
-    <p>Iratkozz fel hírlevelünkre, hogy értesítést kapj a téged érdeklő cikkekről!</p>
-    <input
-      #emailInput
-      [(ngModel)]="email"
-      class="emailInput"
-      name="EMAIL"
-      pattern="^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$"
-      placeholder="email@email.com"
-      type="email"
-      value=""
-    />
-    <button
-      (click)="subscribeToNewsletter()"
-      [disabled]="subscribeButtonDisabled"
-      [ngClass]="{ done: subscribeButtonDone }"
-      class="subscribeButton"
-    >
-      <span *ngIf="buttonTextVisible" class="buttonText">Feliratkozás</span>
-      <fa-icon *ngIf="checkMarkVisible" [icon]="['fas', 'check']"></fa-icon>
-      <div *ngIf="spinnerVisible" class="spinner">
-        <div class="bounce1"></div>
-        <div class="bounce2"></div>
-        <div class="bounce3"></div>
-      </div>
-    </button>
+  <div class="footerblock ekreta">
+    <h4>KRÉTA</h4>
+    <ul>
```

Comment:
Please make the list items have even space between the separating lines

Reply:
Done in cd40c0f